src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentFixture.php

```diff
@@ -24,6 +24,17 @@
 use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
+trigger_deprecation(
+    'sylius/core-bundle',
+    '1.13',
+    'The "%s" class is deprecated and will be removed in Sylius 2.0. Use "%s" instead.',
+    PaymentFixture::class,
```

Comment:
Will it be a BC to make this class extend a new `PaymentFixture` class? They are identical, why not remove code duplication.

Reply:
@jakubtobiasz came up with a practical idea